product-mini/platforms/linux-sgx/enclave-sample/App/App.cpp

```diff
@@ -836,7 +916,7 @@ wamr_pal_create_process(struct wamr_pal_create_process_args *args)
     int stdoutfd = -1;
     int stderrfd = -1;
 
-    int argc = 2;
+    const int argc = 2;
```

Comment:
I think it's a good idea(at least it's no harm) to add this keyword, my IDE identified it as an error otherwise, even though it's correct from the compiler's point of view.